src/main/java/de/helmholtz/marketplace/webappserver/controller/UtilityController.java

```diff
@@ -0,0 +1,42 @@
+package de.helmholtz.marketplace.webappserver.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClient;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientService;
+import org.springframework.security.oauth2.client.authentication.OAuth2AuthenticationToken;
+import org.springframework.security.web.csrf.CsrfToken;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+
+@RestController
+public class UtilityController
+{
+    @Autowired
+    private OAuth2AuthorizedClientService authorizedClientService;
+
+    @GetMapping("/token")
```

Comment:
What is best practice here? Without having much experience here, I would instinctively tend towards defining path fragments in a central static class or in a configuration file. Or is this something normally not done?

Reply:
My question was along the lines of thgis question: https://stackoverflow.com/questions/11741010/should-mapping-value-be-declared-in-a-constant-or-as-an-enum
... but it has probably been resolved reading the mentioned thread

Reply:
I read a little bit further - the initial question was if a enum could be used which does not seem to be possible. The use of a constants class on the othe hand seems to be possible and is recommended by one of the comments. So what I had in mind is to use something like `Constants.MAPPING_TOKEN` instead of `/token` directly.